typescript/baseDAO/src/api.ts

```diff
@@ -186,3 +181,22 @@ export class BaseDAOContract {
       contract => contract.methods.unfreeze(arg));
   }
 }
+
+export class RegistryDAOContract extends BaseDAOContract {
```

Comment:
Shouldn't there be one for `trivialDAO` too?

Reply:
Should be? I think we didn't have one because `trivialDAO` was never meant for real use.

Reply:
:thinking: fair enough, that's a good point.

never mind then